src/System.Runtime.Extensions/src/System/IO/Path.cs

```diff
@@ -49,13 +49,13 @@ public static class Path
 #endif // !PLATFORM_UNIX
 
         // Platform specific alternate directory separator character.  
-        // This is backslash ('\') on Unix, and slash ('/') on Windows 
-        // and MacOS.
+        // This is slash ('/') on Windows and MacOS.
```

Comment:
This comment is out of date, and incorrect.  Probably related to the ':' on Mac above.  It should be cleaned up at the same time.